src/lib/components/particles/ad-slot/stories.module.scss

```diff
@@ -0,0 +1,22 @@
+.adSlotContainer {
+  border: 1px solid #e0e0e0;
+
+  &::before {
+    content: 'Advertisement';
+  display: block;
+  position: relative;
+  font-family: GuardianTextSans, "Guardian Text Sans Web", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
+  font-size: 0.75rem;
```

Comment:
rem sizes behave unexpectedly on the apps unfortunately, which is why we use px sizes